packages/itwin/measure-tools/src/api/SheetMeasurementHelper.ts

```diff
@@ -6,17 +6,36 @@
 import { ColorDef, QueryBinder } from "@itwin/core-common";
 import type { DecorateContext, GraphicBuilder, IModelConnection } from "@itwin/core-frontend";
 import { GraphicType } from "@itwin/core-frontend";
-import type { Point3d } from "@itwin/core-geometry";
+import { Point3d} from "@itwin/core-geometry";
+import { Transform } from "@itwin/core-geometry";
 import { Point2d } from "@itwin/core-geometry";
 import type { DrawingMetadata } from "./Measurement";
 
 export namespace SheetMeasurementsHelper {
 
+  export enum DrawingType {
```

Comment:
I guess this enum maps to an existing enum somewhere else? Please add some information so we know where it comes from.